keyvaluestore/buffermanager.go

```diff
@@ -0,0 +1,167 @@
+package keyvaluestore
+
+type BufferManager interface {
+	Get(nodeId uint64) (*node, error)
+	Put(node *node) error
+	Flush() error
+}
+
+type BufferEntry struct {
+	node                  *node
+	lastAccessTime        uint64
```

Comment:
The last access time is tracked but never really used. Can probably be removed.

Reply:
You are right. I will remove that and the `nextAccessTimestamp` as well. Good catch.